### ISSUE TITLE
Add README about the the SWD is measuring and guidance on using it

### ISSUE THREAD

**Issue by mrchrisadams** [thegreenwebfoundation/co2.js#60]
The SWD model combines work form one paper, and combines it with figures from the IEA for carbon intensity in 2019.

### Using an attributional model to work out the share of global energy usage based on the share of data transferred to use a website

> The Sustainable Web Design takes an observable proxy for the use of digital services  like websites - ***data transferred in Gigabytes (Gb),*** and then draws primarily on a peer reviewed paper by Anders Andrae, *[[New perspectives on internet electricity use in 2030](https://pisrt.org/psr-press/journals/easl-vol-3-issue-2-2020/new-perspectives-on-internet-electricity-use-in-2030/)](https://pisrt.org/psr-press/journals/easl-vol-3-issue-2-2020/new-perspectives-on-internet-electricity-use-in-2030/)*, to convert this figure to ***kilowatt hours of electricity.***
>
> In the paper mentioned above, the authors have conducted an extensive analysis, incorporating data from a variety of academic and industry sources, to generate:
>
> 1. global numbers for total energy use by the ICT sector - around ***1988 Terawatt hours***
> 2. the total network transfer to users, that the sector enables - around **2444 exabytes**
>
> Dividing the top by the bottom, the paper presents a ***kilowatt hours per gigabyte of data transfer*** metric.

I've added a few diagrams here to make it easier to visualise:

https://docs.google.com/presentation/d/12rN3vbhmefFEIaFHSIJ8wjfV0PZpQBPf9T-93WUpCOg/edit#slide=id.p

I'm sharing this from a recent bit of research I'm allowed to share when we explored using this model for understanding the emissions from a website:

### Being explicit about the energy usage and network numbers.

This is from the digital sustainability page:

> *   We’ve made every attempt to ground this work in the latest scientific data available to us. 
> *   Understanding that these are estimates, we rounded up numbers to account for a margin of error.
> *   New studies will inevitably get us closer to specific and accurate data; we’ll adjust our calculations as we learn more.

Source: [Calculating Digital Emissions - Sustainable Web Design](https://sustainablewebdesign.org/calculating-digital-emissions/)

With this in mind, it seems in the spirit of the work to find the most accurate, update to numbers we can if they are available.

While finding energy usage numbers for the ICT are hard, you can look to the [report that Vlad Coroama wrote last summer][vlad](https://www.vs.inf.ethz.ch/publ/bibtex.html?file=../publ/papers/Coroama2021_InternetEnergy), where he points to Cisco Systems's Visual Networking reports as some of the best data publicly available.

> According to (Cisco 2020), the worldwide Internet data traffic for 2020 was around 254 EB/month,corresponding to 3048 EB (or 3.05 zettabytes, ZB) yearly. 

Source: [Investigating the Inconsistencies among Energy and Energy Intensity Estimates of the Internet. Metrics and Harmonising Values](https://www.vs.inf.ethz.ch/publ/bibtex.html?file=../publ/papers/Coroama2021_InternetEnergy)

This figure is 3048 compared to 2444, although whether they're measuring the same thing and it's sensible to swap one out in favour of the other is worth discussing.

### Being explicit about the carbon intensity numbers

Here's another bit from a recent report I wrote exploring this:

> The carbon intensity of electricity can change dramatically depending on where or when it is used, and historically, as the cost of renewable energy has come down the global carbon intensity of electricity has fallen each year.
>
> However, 2021 has bucked this trend, and a number of confounding factors, from lower than average wind in some regions, to upticks in energy demand as the world recovers the pandemic have led to carbon intensity being about 1% higher than the global average in 2020.
>
> Here, we had to make a judgement - while the Sustainable Web Design model was first published in in 2021, the carbon intensity numbers used a global carbon intensity 476grams of CO2 per kilowatt hour of electricity - a figure from 2019. 
>
> If we followed the Sustainable Web Design model *exactly*, would mean we would be applying a figure for electricity usage in 2019, to the year of 2021.
>
> This would represent a difference of about 200 kilogrammes of CO2, or about 8% of the final figure for both sites.
>
> For this analysis we used the global carbon intensity figures for the IEA for 2021 of 440 grams per kilowatt hour of CO2.

It would be good to contribute these numbers back, as they have a meaningful difference on the final numbers in most cases.

**Comment by drydenwilliams** [comment]
Thanks for your comments and write-up @mrchrisadams. To summaries you points you propose the change of: 
- Update total network transfer to 3048 EB
- Update global intensity to 440 grams/kWh
Hope I've understood that correctly?

**Comment by mrchrisadams** [comment]
Yes for the carbon intensity (I'd be specific about carbon intensity, as there are other things you have intensity for like water usage, and so on).

I'm not actually sure about the network transfer one yet - for carbon intensity, you're comparing the numbers from the same source (the IEA), just from later dates. 

For the network transfer, you'd be comparing the transfer number in the paper for the transfer number from a different source.

The idea of what counts as "network" might be different, and I can't remember off the top of my head.

This matters because in some cases people count routers in homes as end user devices, and in some studies they count as part of the network - and if you're swapping a value out, it's important to know that the value you're swapping in is comparable.

I opened this issue here mainly as a bookmark to ask someone - I think I'll try on the ICT 4 sustainability list, as I think Vlad Coroama is on that list would be the logical person to ask.

Alternatively, someone else could read the paper I linked to and add their two cents - I don't have the time right now to read it closely to pull the data out.

**Comment by mrchrisadams** [comment]
OK, slight change of plan.

Rather than the IEA data for carbon intensity, I think using the data from Ember is wiser, as it's very open, the numbers are comparable.

![Screenshot 2022-03-30 at 11 16 53](https://user-images.githubusercontent.com/17906/160797057-c31b06d2-5463-4791-9432-528e471b0424.png)

Because the licensing is open, we could _totally_ have by country figures updated on an annual basis here.

**Comment by mrchrisadams** [comment]
Closing this with #65. I think replacing the modelled figure of 2444 exabytes with the larger one of 3048 exabytes from Cisco's latest report is one we're better off delaying til we've had a chance to check that they refer to the same idea of what the 'network' entails.
